scripts/img_check.sh

```diff
@@ -351,27 +351,19 @@ function checkFirewall {
         ((WARN++))
       fi
     elif [[ $OS == "CentOS Linux" ]]; then
-      if [ -f /usr/lib/systemd/system/csf.service ]; then
+      if [ -f /etc/csf/csf.conf ]; then
         fw="csf"
-        if [[ $(systemctl status $fw >/dev/null 2>&1) ]]; then
-          
-        FW_VER="\e[32m[PASS]\e[0m Firewall service (${fw}) is active\n"
-        ((PASS++))
-        elif cmdExists "firewall-cmd"; then
-          if [[ $(systemctl is-active firewalld >/dev/null 2>&1 && echo 1 || echo 0) ]]; then
-           FW_VER="\e[32m[PASS]\e[0m Firewall service (${fw}) is active\n"
+        csfa=$(csf -l |head -1 |sed "s/ .*//")
+        if [[ $csfa == "iptables" ]]; then
+          FW_VER="\e[32m[PASS]\e[0m Firewall service (${fw}) is active\n"
           ((PASS++))
-          else
-            FW_VER="\e[93m[WARN]\e[0m No firewall is configured. Ensure ${fw} is installed and configured\n"
-          ((WARN++))
-          fi
         else
           FW_VER="\e[93m[WARN]\e[0m No firewall is configured. Ensure ${fw} is installed and configured\n"
         ((WARN++))
```

Comment:
If we hit this else statement we know `csf` is install but not configured, however the warning message says no firewall is configured. Suggest to state `csf is installed but not active as a firewall.` or something to that effect.